### PR TITLE
Changed procedure from AzureRm to Az module

### PR DESCRIPTION
**Pull request by get-itips**
Proposing the procedure to reflect the use of the new Az module

@KCCross Please review

cc @mypil 

https://github.com/MicrosoftDocs/microsoft-365-docs/issues/654

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/controlling-your-data-using-customer-key.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ To set up Customer Key you will complete these tasks. The rest of this topic pro
   
 **In Azure and Microsoft FastTrack:**
   
-You will complete most of these tasks by remotely connecting to Azure PowerShell. For best results, use version 4.4.0 or later of Azure PowerShell.
+You will complete most of these tasks by remotely connecting to Azure PowerShell. For best results, use version 3.0 or later of Azure PowerShell Az module, for more information, see the [Introducing the new Azure PowerShell Az module](https://docs.microsoft.com/powershell/azure/new-azureps-module-az?view=azps-3.0.0).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/controlling-your-data-using-customer-key.md]:
```suggestion
You will complete most of these tasks by remotely connecting to Azure PowerShell. For best results, use version 3.0 or later of Azure PowerShell Az module. For more information, see [Introducing the new Azure PowerShell Az module](https://docs.microsoft.com/powershell/azure/new-azureps-module-az?view=azps-3.0.0).
```

File: microsoft-365/compliance/controlling-your-data-using-customer-key.md [hunk 2]
```diff
@@ -193,26 +193,26 @@ For each key vault, you will need to define three separate sets of permissions f
...
     
-- If you intend to use Customer Key with Exchange Online and Skype for Business, you need to give permission to Office 365 to use the key vault on behalf of Exchange Online and Skype for Business. Likewise, if you intend to use Customer Key with SharePoint Online and OneDrive for Business, you need to add permission for the Office 365 to use the key vault on behalf of SharePoint Online and OneDrive for Business. To give permission to Office 365, run the **Set-AzureRmKeyVaultAccessPolicy** cmdlet using the following syntax: 
+- If you intend to use Customer Key with Exchange Online and Skype for Business, you need to give permission to Office 365 to use the key vault on behalf of Exchange Online and Skype for Business. Likewise, if you intend to use Customer Key with SharePoint Online and OneDrive for Business, you need to add permission for the Office 365 to use the key vault on behalf of SharePoint Online and OneDrive for Business. To give permission to Office 365, run the **Set-AzKeyVaultAccessPolicy** cmdlet using the following syntax:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/controlling-your-data-using-customer-key.md]:
```suggestion
- If you intend to use Customer Key with Exchange Online and Skype for Business, you need to give permission to Office 365 to use the key vault on behalf of Exchange Online and Skype for Business. Likewise, if you intend to use Customer Key with SharePoint Online and OneDrive for Business, you need to add permission for Office 365 to use the key vault on behalf of SharePoint Online and OneDrive for Business. To give permission to Office 365, run the **Set-AzKeyVaultAccessPolicy** cmdlet using the following syntax: 
```

File: microsoft-365/compliance/controlling-your-data-using-customer-key.md [hunk 3]
```diff
@@ -346,46 +346,46 @@ Performing validation before using keys in a DEP is optional, but highly recomme
...
 For example, for SharePoint Online and OneDrive for Business:
   
 ```
-Set-AzureRmKeyVaultAccessPolicy -VaultName Contoso-O365SP-NA-VaultA1 
+Set-AzKeyVaultAccessPolicy -VaultName Contoso-O365SP-NA-VaultA1 
 -PermissionsToKeys wrapKey,unwrapKey,get -ServicePrincipalName TBD
 ```
 
-To verify that an expiration date is not set for your keys run the [Get-AzureKeyVaultKey](https://docs.microsoft.com/powershell/module/AzureRM.KeyVault/Get-AzureKeyVaultKey) cmdlet as follows: 
+To verify that an expiration date is not set for your keys run the [Get-AzKeyVaultKey](https://docs.microsoft.com/powershell/module/az.keyvault/get-azkeyvaultkey) cmdlet as follows:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/controlling-your-data-using-customer-key.md]:
```suggestion
To verify that an expiration date is not set for your keys, run the [Get-AzKeyVaultKey](https://docs.microsoft.com/powershell/module/az.keyvault/get-azkeyvaultkey) cmdlet as follows: 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edits

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review based on issue #654. 

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@KCCross - We would like to follow up on this. Can you please assist to review and approve this PR for merging?
@LauraWi - Would you be able to help merge this PR?

Thank you.

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Any updates on this?

Thank you.

**Comment by get-itips** [comment]
It seems that the affected article has been removed and replaced with https://docs.microsoft.com/en-us/microsoft-365/compliance/customer-key-overview

**Comment by KCCross** [comment]
Apologies for the late comment here - The entire content area was being overhauled for the last several months. This delayed fixes to the existing content that weren't p0 breakfix. Since the Customer Key content is sensitive in nature, I found it prudent to wait until the entire new content set was available before closing these. Thank you for your understanding. I look forward to your feedback on the new content.

**Pull request closed without merging**